docs/admin/etcd.md

```diff
@@ -10,7 +10,7 @@ objects.
 
 ## Configuration: high-level goals
 
-Access Control: give *only* kube-apiserver read/write access to etcd. You do not
```

Comment:
"read/write" is a standard convention for saying "read and write" so I would leave that unchanged.